delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/marshalling/RdfRejectionHandler.scala

```diff
@@ -323,10 +323,10 @@ object RdfRejectionHandler {
     HttpResponseFields(_ => StatusCodes.BadRequest)
 
   implicit private[routes] def compactFromCirceRejection[A <: Rejection: Encoder.AsObject]: JsonLdEncoder[A] =
-    JsonLdEncoder.fromCirce(id = bnode, iriContext = contexts.error)
+    JsonLdEncoder.computeFromCirce(id = bnode, ctx = ContextValue(contexts.error))
```

Comment:
We could have an overloaded computeFromCirce method that would take care of introducing the context value

Reply:
There were too many overloads, I found it more annoying than actually helping

Reply:
The contexts defined in the Vocabulary could be a ContextValue if we never use them as a Iri

Reply:
They are used as [iris too](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/DeltaModule.scala#L43).